example.php

```diff
@@ -6,9 +6,16 @@
   'access_id' => '<access-id>',
   'secret_key' => '<secret-id>',
   'user_id' => '<user-id>',
-  'url_base' => "http://api.integ.nifty.co.ke/api/v1",
+  'api_base' => "http://api.integ.nifty.co.ke/api/v1",
 );
 
+// Till details to be shared.
```

Comment:
Change this to use your till details. Be careful to use the correct platform details (integ vs prod)